api/v3/Job.php

```diff
@@ -140,6 +140,11 @@ function _civicrm_api3_job_execute_spec(&$params) {
  *   API Result Array
  */
 function civicrm_api3_job_geocode($params) {
+  $version = $params['version'] ?? NULL;
+  if ($version) {
+    // We are in api v3, so version has already done it's job.
+    unset($params['version']);
+  }
```

Comment:
`unset()` doesn't care if the variable exists already, so I think this could be simplified as:
```suggestion
  // We are in api v3, so version has already done it's job.
  unset($params['version']);
```